qualtran/bloqs/arithmetic/subtraction.py

```diff
@@ -167,3 +167,84 @@ def _sub_diff_size_regs() -> Subtract:
 _SUB_DOC = BloqDocSpec(
     bloq_cls=Subtract, examples=[_sub_symb, _sub_small, _sub_large, _sub_diff_size_regs]
 )
+
+
+@frozen
+class SubtractFrom(Bloq):
+    """A version of `Subtract` that leaves behind the value being subtracted.
```

Comment:
This phrasing seems a bit confusing to me, perhaps you can state it more concretely?